### DISCUSSION TITLE
Animatediff with AMD love

### DISCUSSION THREAD

**Discussion by FrakerKill** [lshqqytiger/stable-diffusion-webui-directml#305] [General]
Anyone tried to install animatediff automatic 1111 extension? With RX6600 I can't go over 320x320px for 16frames 8fps. Share tests and opinions.

**Comment by Amin456789** [comment]
hmm u can use lyriel 1.6 model, it can generate 256x256 images with very high quality, so all u have to do is generate 320x256 256x256 or something and by using another ai app named flowframes to get the frames, upscale them with realsr and put them back in a gif with flowframes again.
my question is, did u converted animatediff model to onnx too? or just used it's safetensor? i am a cpu user and i wanna try it out
